src/OpenLoco/src/Windows/BuildVehicle.cpp

```diff
@@ -1446,7 +1445,7 @@ namespace OpenLoco::Ui::Windows::BuildVehicle
     }
 
     // 0x4B7741
-    static void drawVehicleOverview(Gfx::RenderTarget* rt, int16_t vehicleTypeIdx, CompanyId company, uint8_t eax, uint8_t esi, Ui::Point offset)
+    void drawVehicleOverview(Gfx::RenderTarget* rt, int16_t vehicleTypeIdx, CompanyId company, uint8_t eax, uint8_t esi, Ui::Point offset)
```

Comment:
Not a fan of having to exposing this for use out of the window, but then again this is one massive function to implement. It'll have to do for now.

Reply:
Yeah it needs to be moved to somewhere else. Just not sure where.